packages/@expo/cli/src/start/doctor/typescript/TypeScriptProjectPrerequisite.ts

```diff
@@ -126,24 +126,25 @@ export class TypeScriptProjectPrerequisite extends ProjectPrerequisite<boolean>
 
   /** Return the first TypeScript file in the project. */
   async _queryFirstTypeScriptFileAsync(): Promise<null | string> {
-    const results = await wrapGlobWithTimeout(
-      () =>
-        // TODO(Bacon): Use `everyMatch` since a bug causes `anyMatch` to return inaccurate results when used multiple times.
-        everyMatchAsync('**/*.@(ts|tsx)', {
-          cwd: this.projectRoot,
-          ignore: [
-            '**/@(Carthage|Pods|node_modules)/**',
-            '**/*.d.ts',
-            '@(ios|android|web|web-build|dist)/**',
-          ],
-        }),
-      5000
-    );
-
-    if (results === false) {
-      return null;
+    try {
+      const results = await everyMatchAsync('**/*.@(ts|tsx)', {
```

Comment:
```suggestion
      // TODO(Bacon): Use `everyMatch` since a bug causes `anyMatch` to return inaccurate results when used multiple times.
      const results = await everyMatchAsync('**/*.@(ts|tsx)', {
```